pkg/imagestore/imagestore.go

```diff
@@ -226,6 +226,9 @@ func (s *rhcosStore) Populate(ctx context.Context) error {
 		imageVersion := imageInfo["version"]
 		arch := imageInfo["cpu_architecture"]
 
+		if arch == "s390x" {
+			continue
```

Comment:
Do you think it's worth adding a comment here about the reason we skip this arch? I.e. maybe a reference to the issue preventing the extracting iso.

Reply:
Sure, I'll add a comment and explain it more thoroughly in the commit message.